failsafe-controller/src/main/java/net/jodah/failsafe/package-info.java

```diff
@@ -0,0 +1,445 @@
+ *     <td>Yes</td>
+ *   </tr>
+ *   <tr>
+ *     <td>Throw or Return</td>
+ *     <td>Simulates a task that returns a specific value or throws a specific exception. Exceptions
+ *         can be configured with actual objects or with class names. In the later case, an exception
+ *         is instantiated using:
+ *         <ol>
+ *           <li>a constructor that accepts a string message; or
+ *           <li>a constructor that accepts a string message and a cause exception; or
+ *           <li>a constructor that accepts a cause exception; or
+ *           <li>a default constructor
+ *         </ol>
+ *         <p>No matter how the exception is created (passed in or instantiated from a class name),
+ *         its stack trace is filled in at the time it is re-thrown.
+ *         <p>Values and exceptions can be intermix. Values that are instance of
+ *         {@link java.lang.Throwable} and classes that extends {@link java.lang.Throwable} will be
+ *         instantiated and thrown out</td>
+ *     <td>0, 1 or many
+ *         <p>depending on how many different values/exceptions the action was configured with (one
+ *         per attempt)</td>
+ *     <td>Yes if there are any configured values/exceptions left to be returned or thrown
+ *         <p>No if no values/exceptions were configured</td>
+ *     <td>Yes</td>
+ *   </tr>
+ *   <tr>
+ *     <td>Interrupt</td>
+ *     <td>Simulate a thread interruption by both raising the interrupted flag of the current thread
+ *         and throwing back an {@link java.lang.InterruptedException}</td>
+ *     <td>1</td>
+ *     <td>Yes</td>
+ *     <td>Yes</td>
+ *   </tr>
+ *   <tr>
+ *     <td>Notify</td>
+ *     <td>Notifies or raises a particular condition in order to wake up or trigger code that might
+ *         be waiting for this condition</td>
+ *     <td>1</td>
+ *     <td>No</td>
+ *     <td>Yes</td>
+ *   </tr>
+ *   <tr>
+ *     <td>Wait</td>
+ *     <td>Blocks the attempt and waits for a particular condition to be notified or raised if not already
+ *         notified.</td>
+ *     <td>1</td>
+ *     <td>No</td>
+ *     <td>Yes</td>
+ *   </tr>
+ *   <tr>
+ *     <td>Wait to be Cancelled</td>
+ *     <td>Blocks the attempt and waits for the execution to be cancelled via the returned
+ *         {@link java.util.concurrent.Future} from Failsafe if not already cancelled.</td>
+ *     <td>1</td>
+ *     <td>No</td>
+ *     <td>Yes</td>
+ *   </tr>
+ * </table>
+ *
+ * <p>
+ *
+ * <table>
+ *   <tr>
+ *     <th>Customization</th>
+ *     <th>Description</th>
+ *     <th>New Cardinality of the Customized Action</th>
+ *   </tr>
+ *   <tr>
+ *     <td>Only If</td>
+ *     <td>Conditionally invokes the customized action based on a static boolean value provided at
+ *         the time the action is constructed or based on the evaluation of a specified predicate
+ *         at the time an attempt is made. if the result is to <code>false</code>, it will move on
```

Comment:
s/b "If the result is &lt;code&gt;false&lt;/code&gt;" (capitalize "If" and remove "to").